packages/sdk/src/protocols/filter/subscription_manager.ts

```diff
@@ -109,6 +135,11 @@ export class SubscriptionManager implements ISubscription {
         callback
       } as unknown as SubscriptionCallback<IDecodedMessage>;
 
+      // don't handle case of internal content topic
+      if (contentTopic === this.buildLightPushContentTopic()) {
```

Comment:
should this be placed earlier in the code?

Reply:
this should be in this callback unfortunately